packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -11,6 +11,11 @@ const SlidePanelPositions = {
 };
 
 const propTypes = {
+  /**
+   * String that labels the Details Panel for screen readers.
+   */
+  ariaLabel: PropTypes.string,
```

Comment:
prop name is too generic - not sure as a consumer whether this applies to the `main` content area or the `panel` content area. 

Would suggest changing this to `panelAriaLabel`, and if needed - also add a companion `mainAriaLabel`.

Reply:
IMO this should be marked as required in the next MVB of slide-panel since this is required for accessibility. Should we log an issue to make this change on the next MVB?

Reply:
Yes i agree. will go ahead and log an issue.

Reply:
Issue link to update the props to be **Required** in the next MVB : https://github.com/cerner/terra-framework/issues/902